x/programs/wasmlanche/src/mock.rs

```diff
@@ -0,0 +1,57 @@
+#[cfg(feature = "test")]
```

Comment:
separate mock functions into a different file for readability

Reply:
put the `cfg` directive on the `mod` declaration in `lib` instead of having an internal mod (`functions`) here